userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -213,7 +214,33 @@ site's [configuration file][].
 [configuration file]: https://gohugo.io/getting-started/configuration/#configuration-file
 [primary color]: #site-colors
 
+### Styling your brand logo and name
 
+The default Docsy navbar (`.td-navbar`) contains your navbar-brand, consisting
+of the following:
+
+1.  [Your logo][], which is included in the navbar as an inline SVG, styled by
+    `.td-navbar .navbar-brand svg`. For the style details, see [_nav.scss][].
+
+    To ensure your logo displays correctly, you may want to resize it and ensure
+    that it doesn't have height and width attributes so that its size is fully
+    responsive. [Override the default styling][project-styles] of `.td-navbar
+    .navbar-brand svg` or (equivalently) `.td-navbar .navbar-brand__logo` as
+    needed.
+2.  Your brand name, which is the site `title`. If you don't want the brand name
```

Comment:
I'd maybe change this from "brand name" to "project name" -> "brand" suggests a commercial product.

Reply:
I agree with your suggested name change, but that would be a breaking change. The term "brand" has been used in the class name from the start of the Docsy project (in GH), 4 years ago.

Do we want to introduce such a breaking change?

Reply:
Just to be clear, I'm quite ok using the terminology "project logo and name" (as you've suggested here and other comments). That, of course, would not be a breaking change. I'm glad to do that.

The breaking change would be if we change the corresponding class names. I'm ok either way.

Reply:
I think I'd just change the terminology in the docs, we can always add something like "..this is defined in navbar-brand" etc. etc.

Reply:
Done ("brand name" to "project name")